src/components/Dashboard/TodoList.vue

```diff
@@ -1075,6 +1076,8 @@ export default class TodoList extends Mixins(
         correctedFilingId: correction.correctedFilingId,
         // this is only used for external corrections (IA):
         correctedFilingType: this.filingTypeToName(correction.correctedFilingType as FilingTypes),
+        // Future: verify correction.type for either CLIENT OR STAFF
+        type: isEmpty(correction.type) ? null : correction.type,
```

Comment:
`correction.type` is an enum, not an object, so you don't need isEmpty.

In fact, why do anything special here -- can't type = correction.type ?